src/folio_migration_tools/migration_tasks/loans_migrator.py

```diff
@@ -256,10 +275,10 @@ def load_and_validate_legacy_loans(self, loans_reader):
             except ValueError as ve:
                 logging.exception(ve)
         logging.info(
-            f"Done validating {legacy_loan_count} legacy loans with {num_bad} rotten apples"
+            f"Done validating {legacy_loan_count + 1} legacy loans with {num_bad} rotten apples"
```

Comment:
Another way to fix this - that I actually prefer - is to set start=1 in the enumerate call. Never mind now though.